packages/odyssey-react/src/components/Checkbox/Checkbox.module.scss

```
@@ -46,58 +46,58 @@
   &:indeterminate {
     + .label {
       .indicator {
-        color: var(--ColorCheck);
+        color: var(--CheckColor);
       }
     }
   }
 
   &:focus {
     + .label {
       .box {
-        @include outline(var(--ColorFocusOutline), var(--FocusOutlineWidth));
+        @include outline(var(--FocusOutlineColor), var(--FocusOutlineWidth));
 
-        border-color: var(--ColorBorderFocus);
+        border-color: var(--FocusBorderColor);
```

```suggestion
        border-color: var(--BoxFocusBorderColor);
```

I believe we want to include the `Box` component part prefix here and for the focus box shadow styles above too yes?